TOMP-API.yaml

```diff
@@ -1633,6 +1633,14 @@ components:
         operation:
           type: string
           enum: [CANCEL, EXPIRE, DENY, COMMIT]
+        origin:
+          type: string
+          description: This operation can be done on behalf of another party. The 
```

Comment:
The wording in the description is confusing. Since this is not a required field, if it's missing it should be considered as `MP` by default?

Reply:
Hello Can, the default is depending on who it is sending. In the description this is said:
"In case this field is obsolete, it must be assumed that the events the MP is sending, this field should contain "MP". And in case the TO is sending, "TO"."

Reply:
That is still confusing :smile: Using the word `obsolete` is also confusing. Did you mean `missing` or `blank` or `empty` instead?